.github/workflows/docs.yml

```
@@ -26,4 +26,10 @@ jobs:
         pip install .
     - name: Build docs
       run: |
-        cd doc && make install
+        cd doc && make html
+    - name: Deploy Docs
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+	github_token: ${{ secrets.GITHUB_TOKEN }}
+	publish_dir: ./html
```

These tabs should be spaces, to keep things consistent.

And `publish_dir` looks like it's not set correctly.

```suggestion
        github_token: ${{ secrets.GITHUB_TOKEN }}
        publish_dir: doc/_build/html
```